### ISSUE TITLE
Avoid the Deadly TLDR Label With These 3 Easy Writing Fixes

### ISSUE THREAD

**Issue by jinglescode** [jinglescode/jinglescode.github.io#60]
# TL;DR
Make your writing easier for readers

# Key Takeaways

- **Tip #1: Format content into smaller chunks**  So break up that fascinating content into smaller chunks. For instance, if you have a web page that clocks in at 2,500 words, then determine how to divide it into two or three sections. Format each section on a separate web page, with plenty of contextual links connecting the pages to each other. This makes your content more digestible.

- **Tip #2: Use more white space** Writing easy-to-read content means writing short paragraphs (no more than 5–6 lines), varying your sentence length, including subheads to create an outline of sorts, and using plenty of bulleted lists. Ellipses, dashes, indentation, and parentheses give your readers’ eyes some breathing room, too. In other words, avoid being TLDR-zoned when you use plenty of white space. That goes for both digital content and print content.

- **Tip #3: Write a TLDR summary** As a writer, you can place a TLDR summary at the beginning of an article, email, or web page as an introduction. Doing so saves the reader’s time. The summary gives her a reason to keep reading (she wants to know more) or gives her a reason to skip the rest of the content (it’s not pertinent to her).

# Link
https://writingcooperative.com/avoid-the-deadly-tldr-label-with-these-3-easy-writing-fixes-145a38fc90c0
